#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_pwm", double_t, 0, "Minimum PWM value",    50, 0,   255)

exit(gen.generate("turtlebot3_custom", "turtlebot3_custom", "Turtlebot3"))
